entrypoints/components/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+const HOST_AUTH = 'http://localhost:3000'
+//  import.meta.env.VITE_AUTH_HOST_URL
+
+interface FooterProps {
+  isAuthAvailable: boolean
+  isAuthenticated: boolean
+  onLogOut: () => void
+}
+
+const AuthButtons = ({
+  isAuthenticated,
+  onLogOut,
+}: Pick<FooterProps, 'isAuthenticated' | 'onLogOut'>) => (
+  <div className="flex flex-row w-full items-center gap-2">
+    {isAuthenticated ? (
+      <button
+        className="flex hover:underline bg-transparent text-sm font-medium text-primary"
+        onClick={onLogOut}
+      >
+        Log out
+      </button>
+    ) : (
+      <>
+        <a
+          href={`${HOST_AUTH}/login?vpnAuth=true`}
+          target="_blank"
```

Comment:
remember to add `rel=‘noopener noreferrer’` to external links for security reasons, some browsers add it by default, but I think it's better to keep it